core/templates/services/schema-default-value.service.ts

```diff
@@ -152,7 +160,10 @@ export class SchemaDefaultValueService {
         schema.type === SchemaConstants.SCHEMA_TYPE_FLOAT) {
       return 0;
     } else {
-      this.logger.error('Invalid schema: ' + JSON.stringify(schema));
+      if (schema.type === 'invalid') {
+        this.logger.error('Invalid schema: ' + JSON.stringify(schema));
+      }
```

Comment:
I don't understand this. Why we need to add explicit checking if the schema has invalid type?

Reply:
Updated PTAL